frontend/src/components/edition/editionRowComponent.tsx

```diff
@@ -38,8 +39,13 @@ export function EditionRowComponent(props: EditionProps) {
     }
 
     async function deleteEdition() {
-        const response = await editionDelete(edition._links.self.href);
-        if (response.status == StatusCodes.NO_CONTENT) {
+        const response = await editionDelete(edition._links.self.href).catch((request) => {
```

Comment:
using catch here with a callback, in combination with async isn't very conssistent.

Reply:
Well can you give me another way? I simply tried checking the status in an else if, but the problem is that when the status code is 409 it is considered an error, that's why I need the catch

Reply:
I will try using try and catch